contrib/format-esri/pom.xml

```diff
@@ -72,7 +72,7 @@
   <dependency>
     <groupId>org.apache.drill.contrib</groupId>
     <artifactId>drill-udfs</artifactId>
```

Comment:
Is this dependency used in code, or it is required for tests only? If for tests, please specify the corresponding scope for it.
Also, please fix the alignment of `<dependencies>` block, it should be indented with two spaces.

Reply:
@vvysotskyi 
Addressed review comments.  This dependency is only required for a specific unit test.